storage/pebble/error.go

```diff
@@ -0,0 +1,5 @@
+package pebble
+
+import "errors"
+
+var ErrNotBootstrapped = errors.New("pebble database not bootstrapped")
```

Comment:
Would this make sense to be part of `storage/errors`? Just wondering since I guess all the register storage implementation will need this, and also we have to document this error on the storage index interface godoc.